Makefile

```diff
@@ -0,0 +1,120 @@
+.DEFAULT_GOAL	:= compile
+
+#------------------------------------------------------------------------------
+# Variables
+#------------------------------------------------------------------------------
+
+SHELL 		:= /bin/bash
+BINDIR		:= bin
+BUILDDIR    := build
```

Comment:
Nit -- indentation

Reply:
Its GH doing the weird thing.. IT looks correct on the terminal.

Reply:
it's tabs